notices.json

```diff
@@ -1,5 +1,23 @@
 [
-{
+    {
+        "id": "focalboard_1",
+           "conditions": {
+          "audience": "sysadmin",
+          "clientType": "all",
+          "serverVersion": [">5.35"],
+          "displayDate": ">= 2021-06-16T00:00:00Z"
+        },
+        "localizedMessages": {
+          "en": {
+            "title": "Enable Focalboard",
+            "description": "Focalboard integrates project management tools like kanban boards, cards, and tables with Mattermost to help you easily plan projects, manage tasks, and run effective meetings. Enable Focalboard today following the steps in the link below.",
+            "image": "https://user-images.githubusercontent.com/46905241/121941290-ee355280-cd03-11eb-9b9f-f6f524e4103e.gif",
```

Comment:
Hi @esethna or @ogi-m, the image is hosted from [this discussion](https://github.com/mattermost/focalboard/discussions/578). Is there a better place to put images?

Reply:
Hey @chenilim, instructions for images are here: https://github.com/mattermost/notices#opening-a-pr

You can add it in this PR branch to the images folder so it gets merged into the release branch where the URL will be pointing

Reply:
Thanks @esethna! Sorry, I missed that part. Should be corrected now - though we would need to commit this before we can double-check that the image URL is right.